frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -164,6 +171,97 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   }
 };
 
+const baseImageUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
+  const state = getState();
+  if (
+    !hasVMSettingsValueChanged(
+      prevState,
+      state,
+      id,
+      VMSettingsField.OPERATING_SYSTEM,
+      VMSettingsField.FLAVOR,
+      VMSettingsField.WORKLOAD_PROFILE,
+    )
+  ) {
+    return;
+  }
+  if (iGetLoadedCommonData(state, id, VMWizardProps.isProviderImport)) {
+    return;
+  }
+
+  const relevantOptions = iGetRelevantTemplateSelectors(state, id);
+  const iCommonTemplates = iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates);
+  const iTemplate =
```

Comment:
this updater is not acknowledging userTemplate selection. So it can dispatch `VMSettingsField.CLONE_COMMON_BASE_DISK_IMAGE` to be true for such template. 
We should ensure it stays false when using userTemplates

Reply:
done